Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -592,8 +594,20 @@ bool ReflectometryReductionOneAuto::processGroups() {
     alg->setProperty("OutputWorkspaceWavelength", IvsLamName);
 
     // Handle transmission runs
-    if (firstTransG)
-      alg->setProperty("FirstTransmissionRun", firstTransG->getItem(i)->name());
+    auto transmissionRunSum = firstTransG->getItem(0)->name();
+    if (firstTransG) {
+      for (size_t item = 1; item < firstTransG->size(); item++) {
```

Comment:
Do not rely on the transmission input having the same number of entries as the InputWorkspace. Create a for loop outside of the main for loop for this.